hydra-node/src/Hydra/Chain/Direct/Handlers.hs

```diff
@@ -194,12 +194,16 @@ data TimeConversionException = TimeConversionException
   }
   deriving (Eq, Show, Exception)
 
+type GetTimeHandle m = m TimeHandle
```

Comment:
Should we keep this synonym or nuke it?

Reply:
I don't need it, but others might deem it useful. If we keep it, we should also be using it above (where we also have a `m TimeHandle`)